app/services/running_meeting_checker.rb

```diff
@@ -1,22 +1,37 @@
 # frozen_string_literal: true
 
-# Pass the rooms to the service and it will return the rooms with the active and participants attributes
+# Pass the room(s) to the service and it will return the rooms with the active and participants virtual attributes
 class RunningMeetingChecker
   def initialize(rooms:)
     @rooms = rooms
   end
 
   def call
-    active_rooms = BigBlueButtonApi.new.active_meetings
-    active_rooms_hash = {}
+    # If provided an array of rooms, the service will use bbb active_meetings request
+    if @rooms.is_a?(Array)
+      active_rooms = BigBlueButtonApi.new.active_meetings
+      active_rooms_hash = {}
 
-    active_rooms.each do |active_room|
-      active_rooms_hash[active_room[:meetingID]] = active_room[:participantCount]
-    end
+      active_rooms.each do |active_room|
+        active_rooms_hash[active_room[:meetingID]] = active_room[:participantCount]
+      end
 
-    @rooms.each do |room|
-      room.active = active_rooms_hash.key?(room.meeting_id)
-      room.participants = active_rooms_hash[room.meeting_id]
+      @rooms.each do |room|
+        room.active = active_rooms_hash.key?(room.meeting_id)
+        room.participants = active_rooms_hash[room.meeting_id]
+      end
+    else
+      # If provided a single room, the service will use bbb get_meeting_info request
+      # get_meeting_info returns an error if the meeting is NOT active
+      begin
+        active_room = BigBlueButtonApi.new.get_meeting_info(meeting_id: @rooms.meeting_id)
+        @rooms.active = true
+        @rooms.participants = active_room[:participantCount]
+      rescue BigBlueButton::BigBlueButtonException
+        @rooms.active = false
+        @rooms.participants = nil
+      end
```

Comment:
Does `get_meeting_info` throw an exception if the meeting isn't running?

Reply:
It returns message: `A meeting with that ID does not exist, messageKey: notFound`
It doesn't seem to be an error as its return code 200.

Reply:
![image](https://user-images.githubusercontent.com/43917914/192820072-47c2e80e-231b-4080-b74f-36da726b068e.png)